code/renderers/react/src/docs/jsxDecorator.test.tsx

```diff
@@ -155,6 +155,23 @@ describe('renderJsx', () => {
     `);
   });
 
+  it('Suspense', () => {
+    function SuspenseComponent(props: any) {
+      return (
+        <React.Suspense fallback={null}>
+          <div>{props.children}</div>
+        </React.Suspense>
+      );
+    }
+
+    expect(renderJsx(createElement(SuspenseComponent, {}, 'I am Suspense'), {}))
+      .toMatchInlineSnapshot(`
+      <SuspenseComponent>
+        I am Suspense
```

Comment:
Any ideas why the `<div>` is not showing up?

Reply:
Hi @shilman 

I submitted a refactor PR: #19785 to handle other "know symbols", but have some issues on CI, you can check it.

Reply:
Uhh, `div`, sorry, it's my problem, I will add it when I come back my office.

Reply:
Actually It works for new creat-react-app when I added them in bundled code, I will figure out why the test passed with wrong snapshot.